<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>USB Video Adapter</string>
  </property>
  <widget class="DisplayWidget" name="displayWidget">
   <property name="minimumSize">
    <size>
     <width>490</width>
     <height>260</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <addaction name="actionDeviceConnect"/>
    <addaction name="actionDeviceDisconnect"/>
    <addaction name="actionDeviceProperties"/>
    <addaction name="separator"/>
    <addaction name="actionDeviceControlServer"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuViewPalette">
     <property name="title">
      <string>&amp;Palette</string>
     </property>
     <addaction name="actionViewPaletteRedGreenBlueInsert"/>
     <addaction name="actionViewPaletteRedGreenBlue"/>
     <addaction name="actionViewPaletteGrayscale"/>
     <addaction name="actionViewPaletteInsert"/>
    </widget>
    <addaction name="menuViewPalette"/>
    <addaction name="separator"/>
    <addaction name="actionViewScale"/>
    <addaction name="actionViewCrop"/>
    <addaction name="separator"/>
    <addaction name="actionViewFullscreen"/>
    <addaction name="actionViewHideMenuBar"/>
   </widget>
   <widget class="QMenu" name="menuPreset">
    <property name="title">
     <string>Presets</string>
    </property>
    <addaction name="actionPresetHelloWorld40"/>
    <addaction name="actionPresetHelloWorld80"/>
    <addaction name="actionPresetUniformColor"/>
    <addaction name="actionPresetColorBands"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuView"/>
   <addaction name="menuPreset"/>
  </widget>
  <action name="actionDeviceProperties">
   <property name="text">
    <string>&amp;Properties...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="actionViewScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;cale to fit window</string>
   </property>
  </action>
  <action name="actionPresetHelloWorld40">
   <property name="text">
    <string>Hello World (&amp;40x25)</string>
   </property>
  </action>
  <action name="actionPresetHelloWorld80">
   <property name="text">
    <string>Hello World (&amp;80x25)</string>
   </property>
  </action>
  <action name="actionPresetUniformColor">
   <property name="text">
    <string>Uniform &amp;color...</string>
   </property>
  </action>
  <action name="actionDeviceControlServer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable TCP &amp;Server...</string>
   </property>
  </action>
  <action name="actionViewPaletteGrayscale">
   <property name="text">
    <string>&amp;Grayscale</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionViewPaletteRedGreenBlue">
   <property name="text">
    <string>&amp;RGB</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionViewHideMenuBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hide Menu Bar</string>
   </property>
  </action>
  <action name="actionViewPaletteInsert">
   <property name="text">
    <string>&amp;Insert</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionPresetColorBands">
   <property name="text">
    <string>Color &amp;Bands</string>
   </property>
  </action>
  <action name="actionViewCrop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Crop &amp;Margins</string>
   </property>
  </action>
  <action name="actionDeviceConnect">
   <property name="text">
    <string>&amp;Connect...</string>
   </property>
  </action>
  <action name="actionDeviceDisconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionViewFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
  </action>
  <action name="actionViewPaletteRedGreenBlueInsert">
   <property name="text">
    <string>RGB+Insert</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayWidget</class>
   <extends>QWidget</extends>
   <header>DisplayWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionViewHideMenuBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>menuBar</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
